hwi/oauth-bundle/0.6/config/packages/hwi_oauth.yaml

```diff
@@ -0,0 +1,29 @@
+hwi_oauth:
+    # list of names of the firewalls in which this bundle is active, this setting MUST be set
+    firewall_names: [secured_area]
+
+    # an optional setting to configure a query string parameter which can be used to redirect
+    # the user after authentication, e.g. /connect/facebook?_destination=/my/destination will
+    # redirect the user to /my/destination after facebook authenticates them.  If this is not
```

Comment:
Do we really need all these comments and configuration also as a comment? I would like to see a link to the documentation instead. 

Also, remove all "inactive" configuration